homeassistant/components/icloud/translations/en.json

```diff
@@ -29,7 +29,8 @@
                 "data": {
                     "password": "Password",
                     "username": "Email",
-                    "with_family": "With family"
+                    "with_family": "With family",
+                    "gps_accuracy_threshold": "Discard inaccurate location updates threshold (meters)"
```

Comment:
Make sure this is also added to `strings.json`.

Reply:
Done!